src/Lex.x

```diff
@@ -12,13 +14,12 @@ $space = [\ \t\f\v\r]
 
 tokens :-
 
-  -- ($space* \n)+       { mk BREAK }
-  -- $space+             ;
   $white+                               ;
 
   -- reserved words
   behaviour                             { mk BEHAVIOUR }
   behavior                              { mk BEHAVIOUR }
+  constructor                           { mk CONSTRUCTOR }
```

Comment:
I had this lexeme before, but I removed it because I didn't like how it made me unable to write specs:
```
behaviour constructor of A
interface contructor()
```
and I would have to make up some other name for this behaviour. One could also skip its name and just call it a constructor of course, but that's why I removed it before

Reply:
Hm, you're right we should find a different name because "constructor" is a common name. I do think behaviours should be syntactically distinguished from constructors though, since they have different semantics. Should be something like
```
constructor_keyword constructor of Contract
interface ...
```

Reply:
we could also disallow custom naming of the `constructor` section at all and just do:
```
constructor of Contract
interface ...
```

Reply:
i chose `creator` for now, and removed their name field